PC02_Graffiti_HEISNER.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: YOUR NAME
+May 29, 2020
+'''
+
```

Comment:
add a description for what the code does
```suggestion
#this code turns jeff bezos into a clown
```